nexus/src/app/background/tasks/decommissioned_disk_cleaner.rs

```diff
@@ -0,0 +1,458 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Cleans up old database state from decommissioned disks.
+//!
+//! This cannot happen at decommissioning time, because it depends
+//! on region replacement, which happens in the backgorund.
```

Comment:
1) and snapshot replacement
2) typo: backgorund

Reply:
Fixed